.github/workflows/ci_nightly.yml

```diff
@@ -0,0 +1,97 @@
+# This CI workflow is triggered by:
+#   - scheduled run
+#
+# In the scheduled run, we run all targets from amdgpu_family_matrix.py and amdgpu_family_matrix_xfail.py
+# As some of these builds are xfail, we allow errors to occur with `continue-on-error`, where the job will fail but the workflow is green
+
+name: CI Nigthly
```

Comment:
Oops, just noticed a typo here.

```diff
-name: CI Nigthly
+name: CI Nightly
```


I think we should also add a status badge to the root README next to `CI`: https://github.com/ROCm/TheRock/blob/28a8bc6665cbf88579896ac7519c71b45671396c/README.md?plain=1#L3